aws_marketplace/listing_torchserve_models/listing-torchserve-models.ipynb

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
<p>Please do not upgrade people's kernel with SageMaker v2 as this will make all other v1 notebooks fail.</p><p>If you require v2, please upgrade to it, then roll it back in the notebook's cleanup section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1573/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
We do need SageMaker v2 for invoking the endpoints. I will add a cleanup section in the end to roll back to SageMaker and mark it as optional.

Could you kindly advise a suggested code to rollback to SageMaker v1 at the end?